skabase/SKAMaster/SKAMaster.xmi

```diff
@@ -66,24 +66,6 @@
       </argout>
       <status abstract="true" inherited="true" concrete="true"/>
     </commands>
```

Comment:
The `MetricList` should have been removed as well.

As mentioned previously, if the XMI files are changed, you need to to purge the XMI changes using [this script](ska-telescope/lmc-base-classes/blob/master/scripts/purge_xmi_tree.py) and then regenerate all the Python files (using the modified POGO executable).  That should solve these kinds of problems.  This PR must include changes to all the base class and reference element files because they all inherit from SKABaseDevice which has now changed.

All traces of MetricsList, GetMetrics and ToJson should be removed from the whole repo.